plugin/trino-delta-lake/src/test/java/io/trino/plugin/deltalake/BaseDeltaLakeConnectorSmokeTest.java

```diff
@@ -1746,6 +1750,26 @@ public void testVacuum()
             assertThat(getActiveFiles(tableName)).isEqualTo(updatedFiles);
             // old files should be cleaned up
             assertThat(getAllDataFilesFromTableDirectory(tableName)).isEqualTo(updatedFiles);
+            // operations should be logged
+            assertThat(query("SELECT version, operation, MAP_FILTER(operation_parameters, (k, v) -> k <> 'queryId'), operation_metrics FROM \"" + tableName + "$history\"")).matches( """
+                    VALUES\s
+                    (CAST(0 AS BIGINT), CAST('CREATE TABLE AS SELECT' AS VARCHAR), CAST(MAP() AS MAP(VARCHAR, VARCHAR)), CAST(MAP() AS MAP(VARCHAR, VARCHAR))),\s
+                    (1, 'MERGE', MAP(), MAP()),\s
+                    (2, 'VACUUM START', MAP(), MAP(ARRAY['sizeOfDataToDelete', 'numFilesToDelete'], ARRAY['0', '0'])),\s
+                    (3, 'VACUUM END', MAP(ARRAY['status'], ARRAY['COMPLETED']), MAP(ARRAY['numDeletedFiles', 'numVacuumedDirectories'], ARRAY['0', '1'])),\s
+                    (4, 'VACUUM START', MAP(), MAP(ARRAY['sizeOfDataToDelete', 'numFilesToDelete'], ARRAY['5025', '5'])),\s
+                    (5, 'VACUUM END', MAP(ARRAY['status'], ARRAY['COMPLETED']), MAP(ARRAY['numDeletedFiles', 'numVacuumedDirectories'], ARRAY['5', '1']))""");
```

Comment:
Add test when there are more than 1 directory to clean

Reply:
The implementation was not according to the Delta core implementation. Fixed now. It shouuld have already listed more than one directory.